### PR TITLE
Add Bieli et al example for JOSS paper

### PR DESCRIPTION
**Pull request by edejong-caltech**
Bieli et al example now working with fixes to breakup overflow in PySDM repo

### CODE REVIEW

**Comment by slayoo** [comment]
Thank you Emily!
Just updated `test-time-requirements.txt` here so the tests are run with the relevant changes in PySDM

**Comment by slayoo** [comment]
green! :) @edejong-caltech

**Comment by slayoo** [comment]
@edejong-caltech 
- let's try to fix this PR so it does not claim that almost 100 files were modified
- I'd suggest removing the .pdf plot file - the plot is included in the notebook, hence no benefit from storing it also in pdf
- let's remove `os.environ['NUMBA_DISABLE_JIT'] = '1'`, `%load_ext autoreload` and `%autoreload`
- let's add an entry to the README.md file with badge-links to the new example notebook
- let's add nbviewer, Colab and mybinder badges as in other examples
- ditto (i.e. last two points) for the deJong_Mackay_2022 examples

Thanks!

**Comment by edejong-caltech** [comment]
Just took care of the above points @slayoo . Let's see if the checks pass.

**Comment by edejong-caltech** [comment]
See PR816 in PySDM repo for fix to remaining issue

File: PySDM_examples/Shipway_and_Hill_2012/settings.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def drhod_dz(z, rhod):
         self.condensation_rtol_x = condensation.DEFAULTS.rtol_x
         self.condensation_rtol_thd = condensation.DEFAULTS.rtol_thd
         self.condensation_adaptive = True
-        self.coalescence_adaptive = True
+        self.coalescence_adaptive = False
```
Comment (slayoo) [inline PySDM_examples/Shipway_and_Hill_2012/settings.py:77]: @edejong-caltech was it intentional?

Comment (edejong-caltech) [reply PySDM_examples/Shipway_and_Hill_2012/settings.py:77]: Nope

**Pull request merged**